pkg/server/datastore/sqlstore/sqlstore.go

```diff
@@ -0,0 +1,136 @@
+package sqlstore
+
+import (
+	"errors"
+	"fmt"
+
+	"gorm.io/gorm"
+)
+
+const (
+	// MySQL database type
+	MySQL = "mysql"
+	// PostgreSQL database type
+	PostgreSQL = "postgres"
+	// SQLite database type
+	SQLite = "sqlite3"
+)
+
+func OpenDB(ConnectionString string, dbtype string) (db *gorm.DB, err error) {
+	var dialectvar dialect
+
+	switch dbtype {
+	case SQLite:
+		dialectvar = sqliteDB{}
+	case PostgreSQL:
+		dialectvar = postgresDB{}
+	case MySQL:
+		dialectvar = mysqlDB{}
+	default:
+		return nil, errors.New("unsupported database_type: %s" + dbtype)
+	}
+	db, err = dialectvar.connect(ConnectionString)
+	if err != nil {
+		return nil, errors.New("Error connecting to: %s" + ConnectionString)
+	}
+	return db, nil
+}
+
+func CreateOrganizationTableInDB(db *gorm.DB) error {
+	// Creates the Model for the Bridge
+	result := db.AutoMigrate(&Organization{})
+	if result != nil {
+		fmt.Println("storage err: ", result)
+		return result
+	}
+	return nil
+}
+
+func InsertOrg(db *gorm.DB, org Organization) error {
+	// Insert new ORG into the DB
+	err := db.Where(&org).FirstOrCreate(&org).Error
+	if err != nil {
+		fmt.Println("storage err: ", err)
+		return err
+	}
```

Comment:
super nit: this can be simplified like this
```suggestion
	if err := db.Where(&org).FirstOrCreate(&org).Error; err != nil {
		return fmt.Errorf("sqlstorage error: %v", err)
	}
	return nil
```